app/build.gradle

```diff
@@ -81,6 +81,7 @@ dependencies {
 
     implementation 'com.github.ARK-Builders:ark-filepicker:c6d66141c1'
     implementation "androidx.navigation:navigation-compose:2.5.2"
+    implementation 'com.google.code.gson:gson:2.10.1'
```

Comment:
@mdrlzy is this dependency ok? I think we use something different in `arklib-android`

Reply:
Sure

Reply:
btw Schubert has already replaced with `kotlinx-serialization`